Makefile

```diff
@@ -6,6 +6,9 @@
 # TODO: Add in the necessary variables, etc, to make this Makefile work.
 # TODO: Add in multi-arch stuff.
 
+# Section for compatibility with kubebuilder / operator-sdk.
```

Comment:
Not actually sure this is needed... copied from the upgrade guide. Might need to revisit.